data_display/templates/data_display/database_start_page.html

```diff
@@ -14,9 +14,15 @@ <h2>This is the VEEP Database Search Engine</h2>
         <p> You can use this search engine to query through data that VEEP has stored to
         learn more about the club, projects, student demographics
         and other available data stored over the years.</p>
+
         <h3>Filtering the data using the Filter tool</h3>
         <p>When filtering the data enter your filter as 'category,value' where the category corresponds to the
         the category you want to filter by and the value is what you are filtering for. For example: 'project_name,VEEP Database Improvement' will filter by projects for the Data Display project </p>
+        <h4>Category choices (case sensitive) are:</h4>
+        <p>For Students: student_id, name, email, discipline, year, phone, interview_offer, project_name</p>
```

Comment:
Can you do this iteratively using templates? In views.py you can see that you can get column names as a list by calling `<model>.__meta.get_fields(). `

Hard-coding the possible choices like this means we have to spend extra time editing the HTML if we ever change the columns of the tables (which is one of the main functionalities of the website, if you recall way back to last year when we drafted project requirements).